packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
```

Comment:
I personally prefer to use more explicit names, especially since we could add other non-index related actions later on (e.g. delete document, get document, etc.).

Namespacing the action names into something like `elasticsearch.index.optimize`, `elasticsearch.documents/search.query`, etc. is probably better.